coding-concepts.js

```diff
@@ -21,26 +27,26 @@ const greeter = (name) => {
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:`Hello, LEARN Student!`
+// b) Verify and explain: undefined. I don't know why it's returning as undefined.
```

Comment:
Whenever you have a function that is returning undefined, check if your function has a return.  In Javascript, every function has to have a return.